sw-embed/src/styles/sw-embed.css

```diff
@@ -80,32 +80,6 @@ sw-channel h2 {
   margin: 0 0 20px 0;
```

Comment:
You probably want to remove the rule for sw-loading (lines 14-17) from sw-embed.css since that's what provides the background gray that you didn't like. Looks like you removed it from a demo but not the default style file